requirements.txt

```diff
@@ -12,3 +12,4 @@ keyring==8.5.1
 virtualenv
 pluggage
 dockerstache>=0.0.9
+pycrypto==2.6
```

Comment:
IIRC this comes via fabric, so the fix was to update fabric in the venv. Would rather let fabric specify the version, than shadow the requirement here.